src/extension.ts

```diff
@@ -416,6 +419,11 @@ async function checkSettingsHealth(settings: Settings): Promise<boolean> {
     return ok
 }
 
+async function getMachineId(): Promise<string> {
+    const proc = new ChildProcess('hostname', [], { collect: true })
```

Comment:
should probably pass `logging:false`, else this will be noisy on machines that don't have `hostname`.

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/4005